src/main/resources/com/google/api/codegen/nodejs/test.snip

```diff
@@ -48,8 +48,10 @@
       // Mock request
       {@initCodeLines(test.initCode)}
 
-      // Mock response
-      {@initCodeLines(test.mockResponse.initCode)}
+      @if test.hasReturnValue
+        // Mock response
+        {@initCodeLines(test.mockResponse.initCode)}
+      @end
 
```

Comment:
Include this blank line in the `@if/@end`. Otherwise we end up with double-blanks whenever there's no return value.